src/less/InfoBarWidget.less

```diff
@@ -0,0 +1,149 @@
+@import 'mixins';
+
+@spinner-size: 12px;
+
+.ext-wwt-infoBarWidget {
+	position: relative;
+	display: flex;
+	z-index: 1;
+	width: 100%;
+	box-shadow: 0 2px 6px 0 rgba(0, 0, 0, 0.25);
+	padding: 16px;
+	color: #ffffff;
+	.skin-vector & {
+		// This duplicates Vector, making sure that the
+		// bar stops at the edge of the content even though
+		// it is outside the content box itself.
+		.ltr& {
+			margin-left: 11em;
+		}
+		.rtl& {
+			margin-right: 11em;
+		}
+		width: calc( 100% - 11em - 2em );
+	}
+
+	&.ext-wwt-infoBarWidget-sticky {
+		position: fixed;
+		top: 0;
+	}
+
+	.oo-ui-iconElement-icon {
+		width: 1em;
+	}
+
+	.oo-ui-labelElement-label {
+		flex-grow: 2;
+		margin-left: 2em;
+		.rtl & {
+			margin-left: 0;
+			margin-right: 2em;
+		}
+	}
+
+	.ext-wwt-infoBarWidget-close.oo-ui-iconElement {
+		margin: 0 1em;
+		cursor: pointer;
+
+		&.oo-ui-flaggedElement-pending {
+			display: none;
+		}
+	}
+
+	&.oo-ui-flaggedElement-pending {
+		background-color: #2a4b8d;
+
+		.ext-wwt-infoBarWidget-spinner {
+			display: inline-block;
+			width: 50px;
+			white-space: nowrap;
+
+			.ext-wwt-infoBarWidget-spinner-bounce,
+			&:before,
+			&:after {
+				content: '';
+				background-color: #fff;
+				display: block;
+				float: left;
+				margin-top: @spinner-size / 2;
+				width: @spinner-size;
+				height: @spinner-size;
+				border-radius: 100%;
+				.animation( wwtBouncedelay 1600ms ease-in-out -160ms infinite both );
+			}
+
+			&:before {
+				margin-right: 4px;
+				.animation-delay( -330ms );
+			}
+
+			&:after {
+				margin-left: 4px;
+				.animation-delay( 0s );
+			}
+		}
+	}
+
+	&.oo-ui-flaggedElement-err {
+		background-color: #b32424;
+
+		.oo-ui-labelElement-label {
+			margin-left: 2em;
+		}
+	}
+
+	&.oo-ui-flaggedElement-ready {
+		background-color: #3366cc;
+
+		.ext-wwt-infoBarWidget-info.oo-ui-labelWidget {
+			flex-grow: 2;
+			text-align: right;
+
+			.rtl & {
+				text-align: left;
+			}
+		}
+	}
+}
+
+
+@-webkit-keyframes wwtBouncedelay {
+	// 50% equals 800ms
+	0%,
+	50%,
+	100% {
+		-webkit-transform: scale( 0.625 );
+	}
+
+	// equals 320ms
+	20% {
+		opacity: 0.87;
+		-webkit-transform: scale( 1 );
+	}
+}
+
+@-moz-keyframes wwtBouncedelay {
+	0%,
+	50%,
+	100% {
+		-moz-transform: scale( 0.625 );
+	}
+
+	20% {
+		opacity: 0.87;
+		-moz-transform: scale( 1 );
+	}
+}
```

Comment:
`@keyframes` is supported in all modern browsers, even IE 10, so we shouldn't need the `-webkit` and `-moz` prefixes

Reply:
Good point. We're still using that in core, so a part of me wonders why... but in this case, it's probably safe anyways. Removing that.